### DISCUSSION TITLE
about how to unpublish GitHub pages

### DISCUSSION THREAD

**Discussion by bluesky62** [community/community#36129] [Other Feature Feedback, Questions, & Ideas] [answered]
I publish the GitHub page but now this page not showing there so I want to republish how i do please help

**Comment by DSDmark** [comment, marked as answer]
Hey,
so, there are two things that you can do if you want to republish your github-page
### first just go in your repo and click on settings 
- ![Screenshot from 2022-10-14 17-15-55](https://user-images.githubusercontent.com/85744816/195839474-55f14bb7-87e4-4059-ab38-6698fcf04d2c.png)

### and the on environments you can see your git page. In there you can delete that page
![Screenshot from 2022-10-14 17-16-08](https://user-images.githubusercontent.com/85744816/195839596-817d1813-b140-4bfc-bfd9-b59c5b3982f8.png)
![Screenshot from 2022-10-14 17-17-32](https://user-images.githubusercontent.com/85744816/195839741-f43848d2-0854-40c2-b2aa-ec7ffe785ac2.png)


In simpler word 

https://github.com/user/repo-name/settings/environments

### if you want to unpublish your repo then you can do same step in page section in settings

![Screenshot from 2022-10-14 17-23-50](https://user-images.githubusercontent.com/85744816/195840765-74c3a040-f74a-47c5-ae81-27a661470cca.png)
